docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -0,0 +1,43 @@
+get:
+  description:
+    List all running tasks.
+  is: [ secured ]
+  queryParameters:
+    status:
+      description: Filter the list of tasks by status
+      enum: [ TASK_STAGING, TASK_STARTING, TASK_RUNNING, TASK_FINISHED, TASK_FAILED, TASK_KILLED, TASK_LOST, TASK_ERROR ]
```

Comment:
In theory we could report tasks in other states. From an API perspective that could make sense. In order to document the current status I agree.